src/components/OrdersWidgetDemo/index.tsx

```diff
@@ -1,39 +1,37 @@
 import React from 'react'
-import Tabs, { TabItemType, TabThemeType } from 'components/common/Tabs/Tabs'
+import Tabs, { getTabTheme, TabItemInterface } from 'components/common/Tabs/Tabs'
 import { OrdersWidgetDemo as Wrapper } from './OrdersWidgetDemo.styled'
 import { ActiveOrdersContent } from './ActiveOrdersContent'
 
-const tabItems: TabItemType[] = [
+const tabItems: TabItemInterface[] = [
   {
     id: 1,
-    title: 'Active Orders',
+    tab: 'Active Orders: ' + 5,
     content: <ActiveOrdersContent />,
-    count: 5,
   },
   {
     id: 2,
-    title: 'Order History',
+    tab: 'Order History: ' + 10,
```

Comment:
What is the reason for adding a number here?

Reply:
you had `count` before which wasn't used, as well as opinionated. This is to show count and not need a prop inside the `Tab` generic which shouldn't be aware of it. Leave it to the passed in `tab` prop to deal with that

Reply:
Ah got it 👍🏼